spec/unit/system_description_spec.rb

```diff
@@ -197,22 +197,29 @@ class SystemDescriptionFooConfig < Machinery::Object; end
   end
 
   describe "#assert_scopes" do
+    let(:working_description) {
+      create_test_description(
+        name: @name, scopes: ["packages", "repositories", "config_files"]
+      )
+    }
+
     it "checks the system description for completeness" do
-      full_description = create_test_description(name: @name, json: @description)
       [
         ["repositories"],
         ["packages"],
-        ["repositories", "packages"]
+        ["repositories", "packages", "config_files"]
       ].each do |missing|
```

Comment:
Do we really need to test each case? we could just test the case when the required scopes are there and when they are not